build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+p=`pwd`
+for d in $(ls ./cmd); do
```

Comment:
is there a specific reason for choosing `/cmd` as the directory? I haven't seen this before.